src/icons/IconTags.tsx

```diff
@@ -5,67 +5,219 @@ import { IIconSize } from "../utils/types";
 const IconSize = {
   12: (
     <SIconBase width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
-      <path d="M6.10985 6.7636C6.2054 6.85915 6.2054 7.01406 6.10985 7.10961C6.01431 7.20516 5.85939 7.20516 5.76384 7.10961C5.6683 7.01406 5.6683 6.85915 5.76384 6.7636C5.85939 6.66805 6.01431 6.66805 6.10985 6.7636" strokeWidth="1.2" strokeLinecap="round" strokeLinejoin="round"/>
-      <path fillRule="evenodd" clipRule="evenodd" d="M7.80667 4.19333L12.6133 9C13.128 9.51467 13.128 10.3493 12.6133 10.864L9.864 13.6133C9.34933 14.128 8.51467 14.128 8 13.6133L3.19333 8.80667C3.07 8.68333 3 8.51533 3 8.34067V4.65933C3 4.29533 3.29533 4 3.65933 4H7.34133C7.516 4 7.68333 4.06933 7.80667 4.19333V4.19333Z" strokeWidth="1.2" strokeLinecap="round" strokeLinejoin="round"/>
-      <path d="M13 6.66667L8.66267 2.38467C8.41267 2.138 8.07667 2 7.726 2H5" strokeWidth="1.2" strokeLinecap="round" strokeLinejoin="round"/>
+      <path
```

Comment:
@nicolasshiken please avoid these kind of updates that introduces noise to reviews and may cause conflicts with other PRs. This change is not related to the implementation.

Reply:
My bad! I must have changed something and then undone the change, but left the formatting there. I'll be more careful and discard the changes instead next time

Reply:
My bad! I must have changed something and then undone the change, but left the formatting there. I'll be more careful and discard the changes instead next time

Reply:
My bad! I must have changed something and then undone the change, but left the formatting there. I'll be more careful and discard the changes instead next time